app/src/test/java/org/simple/clinic/summary/medicalhistory/MedicalHistorySummaryLogicTest.kt

```diff
@@ -71,30 +71,9 @@ class MedicalHistorySummaryLogicTest {
   private val clock = TestUtcClock()
 
   private val events = PublishSubject.create<UiEvent>()
-  private val effectHandler = MedicalHistorySummaryEffectHandler(
-      schedulers = TrampolineSchedulersProvider(),
-      medicalHistoryRepository = medicalHistoryRepository,
-      userSession = userSession,
-      facilityRepository = facilityRepository,
-      clock = clock,
-      uiActions = ui
-  )
 
   private lateinit var testFixture: MobiusTestFixture<MedicalHistorySummaryModel, MedicalHistorySummaryEvent, MedicalHistorySummaryEffect>
 
-  @Before
-  fun setUp() {
-    val uiRenderer = MedicalHistorySummaryUiRenderer(ui)
-    testFixture = MobiusTestFixture(
-        events = events.ofType(),
-        defaultModel = MedicalHistorySummaryModel.create(patientUuid),
-        init = MedicalHistorySummaryInit(),
-        update = MedicalHistorySummaryUpdate(),
-        effectHandler = effectHandler.build(),
-        modelUpdateListener = uiRenderer::render
-    )
-  }
-
```

Comment:
Why was this moved into `setupController`? As far as I can tell, there's nothing in there which needs to be overriden at a test level, right?

Reply:
I needed to pass the `facility` to the `EffectHandler`. The `facility` is being sent at the test level.

Reply:
Ah, I see it now.